pkg/integration_tests/oneagent_controller_integration_test.go

```diff
@@ -36,4 +37,33 @@ func TestReconcileOneAgent_ReconcileOnEmptyEnvironment(t *testing.T) {
 
 	assert.Equal(t, DefaultTestNamespace, dsActual.Namespace, "wrong namespace")
 	assert.Equal(t, oaName, dsActual.GetObjectMeta().GetName(), "wrong name")
+	assert.Equal(t, corev1.DNSClusterFirst, dsActual.Spec.Template.Spec.DNSPolicy) // default when empty.
```

Comment:
nitpick - remove the comment here as well, please incorporate these into message args etc.

Reply:
Assert messages are used to describe why something failed. I just added it as a reference since it's not obvious what is the default.

Reply:
I would, in that case, nest two tests - one with default behavior, and one without (assuming if you have added that comment, there is no test case demonstrating default behavior?)